apps/flame-defi/app/pool/components/pool-txn-steps.tsx

```diff
@@ -0,0 +1,293 @@
+"use client";
+
+import { TXN_STATUS } from "@repo/flame-types";
+import {
+  BlockLoader,
+  Skeleton,
+  SuccessCheck,
+  TokenIcon,
+} from "@repo/ui/components";
+import { ErrorIcon } from "@repo/ui/icons";
+import { useEvmChainData } from "config";
+import { usePathname } from "next/navigation";
+import { usePoolDetailsContext } from "pool/hooks";
+import {
+  PoolTxnStepsProps,
+  TxnComponentProps,
+  TxnFailedProps,
+  TxnLoaderProps,
+  TxnSuccessProps,
+  TxnType,
+  getTxnType,
+} from "pool/types";
+import { useIntl } from "react-intl";
+
+export const CollectFeeTxnDetails = ({
```

Comment:
Would `DisplayDetails` be immediately understandable? Details for what?

Reply:
We have three pool actions, Collect Fees, Add liquidity, and Remove liquidity. Each have slightly different details that need to be displayed in the confirmation modals. To me `CollectFeeTxnDetails` is a detailed name for what is displayed. `DisplayDetails` is more vague and would be appropriate if we were using one component for displaying all of the transaction details the same for the three actions.

Reply:
What about something like `Summary`, `Overview`, `Breakdown`? Do any of these words make the intent more clear than "details"?